.tekton/docker-build.yaml

```diff
@@ -166,7 +166,7 @@ spec:
       - name: name
         value: prefetch-dependencies-oci-ta
       - name: bundle
-        value: quay.io/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta:0.1@sha256:90e1a4fd2c588f3f3b32d3bc7aa1e29ae0233dd8f976fa0532df508e60a345b3
+        value: quay.io/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta@sha256:593714c3b029ef748a38d6cefea122294df2390172d7694b720be62bb416ff59
```

Comment:
/hold

Tag needs to be there

Reply:
Which tag?

Reply:
The image tag was removed

![image](https://github.com/user-attachments/assets/4e579af9-2ef3-452d-ab0b-43e73e430c18)

Reply:
The tag is back should we merge @dsimansk ?

Reply:
Yes, indeed.